app/assets/src/components/views/compare/SamplesHeatmapVis.jsx

```diff
@@ -81,6 +81,7 @@ class SamplesHeatmapVis extends React.Component {
         // only display colors to positive values
         scaleMin: 0,
         printCaption: this.generateHeatmapCaptions(),
+        sortColumns: this.props.sortSamples === "alpha", // else cluster
```

Comment:
Nit: Can we name this `shouldSortColumns`? The reason being that sortSamples and sortColumns sound very similar, yet one is a boolean and the other is a string.

Reply:
👍

Reply:
👍